wildlifelicensing/apps/applications/views/issue.py

```diff
@@ -115,6 +117,10 @@ def post(self, request, *args, **kwargs):
 
             licence.save()
 
+            licence.variants.clear()
+            for index, avl in enumerate(application.variants.through.objects.all().order_by('order')):
+                WildlifeLicenceVariantLink.objects.create(licence=licence, variant=avl.variant, order=index)
```

Comment:
I'm pretty sure it is not important and that they are the same but `order=avl.order` instead `order=index` (no need of enumerate)